lib/protobuf/protoc/generator/util.ex

```diff
@@ -82,4 +82,10 @@ defmodule Protobuf.Protoc.Generator.Util do
     |> Code.format_string!(locals_without_parens: @locals_without_parens)
     |> IO.iodata_to_binary()
   end
+
+  @spec version() :: String.t()
+  def version do
+    {:ok, value} = :application.get_key(:protobuf, :vsn)
+    value
```

Comment:
`:application.get_key/2` returns a charlist, not a binary, let's do this:

```suggestion
    List.to_string(value)
```